Sources/Test.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+func test() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.swift/blob/b2e7e1278e910d6f86ac35a7fc7e0d96abe2e914/Sources/Test.swift#L3"></span>The 'creationDate' is the require reason api. Please check privacy manifest files.</td>
    </tr>
  </tbody>
</table>